configure.sh

```diff
@@ -164,6 +167,14 @@ build:windows --copt=/w
 # On windows, we never cross compile
 build:windows --distinct_host_configuration=false
 
+build --define=use_fast_cpp_protos=true
+build --define=allow_oversize_protos=true
+
+# Enable using platform specific build settings, except when cross-compiling for
+# mobile platforms.
+build --enable_platform_specific_config
```

Comment:
Nice I've been meaning to do this as well.
Since this will automatically set the correct config depending on the platform, I think we should get rid of [those lines](https://github.com/larq/compute-engine/pull/366/files#diff-735003ab6f5797c8b8bdecbbbb9322a8R207-R211), specifically the `--config=windows` is now unnecessary.